### DISCUSSION TITLE
Script is blocked by uBlock Origin even behind Cloudflare worker

### DISCUSSION THREAD

**Discussion by vitobotta** [plausible/analytics#1758] [Anything Else]
Hi! I have configured a Cloudflare workers to proxy requests to plausible.io using blah.js as name of the script and /foo as event post path. Yet uBlock Origin blocks the script regardless. How does it do that and is there a workaround? Thanks!

**Comment by ukutaht** [comment]
What's the full url of the script?

You should be able to check which rule in uBlock is blocking it to understand better whats going on
